spec/features/admin/settings_spec.rb

```diff
@@ -89,4 +89,49 @@
 
   end
 
-end
+  describe "Skip verification" do
+
+    scenario "deactivate skip verification", :js do
+      Setting["feature.user.skip_verification"] = 'true'
+      setting = Setting.where(key: "feature.user.skip_verification").first
+      admin = create(:administrator).user
+      login_as(admin)
+
+      visit admin_settings_path
+
+      accept_alert do
+        find("#edit_setting_#{setting.id} .button").click
+      end
+
+      logout
+
+      user = create(:user)
+
+      expect(user.sms_verified?).to eq(false)
+
+      Setting["feature.user.skip_verification"] = nil
```

Comment:
Both `Setting["feature.user.skip_verification"] = 'true'` can be moved onto a `before` block, and both `Setting["feature.user.skip_verification"] = nil` into an after block, as I said in the comments about this spec